wizeline/remix-project-lab
# Feat - 87 - Extract Permission Logic

# What does this PR do?

Extracts permission logic from `project.server.ts` file to a separate file. Small bugfixes.

# Where should the reviewer start?

app/models/project.server.ts
app/utils.ts

# How should this be manually tested?

Go to Project page. Make sure that the page works correctly
Go to project members page (contributors) make sure the page works and saves correctly

# Any background context you want to provide?



# What are the relevant tickets?

https://github.com/wizeline/remix-project-lab/issues/87

# Screenshots



# Questions



# Checklist
